mbed_host_tests/host_tests_runner/host_test_default.py

```diff
@@ -74,7 +74,39 @@ def __init__(self, options):
 
         DefaultTestSelectorBase.__init__(self, options)
 
+    def is_host_test_obj_compatible(self, obj_instance):
+        """! Check if host test object loaded is actually host test class
+             derived from 'mbed_host_tests.BaseHostTest()'
+             Additionaly if host test class implements custom ctor it should
+             call BaseHostTest().__Init__()
+        @param obj_instance Instance of host test derived class
+        @return True if obj_instance is derived from mbed_host_tests.BaseHostTest()
+                and BaseHostTest.__init__() was called, else return False
+        """
+        result = False
+        if obj_instance:
+            result = True
+            self.logger.prn_inf("host test class: '%s'"% obj_instance.__class__)
+
+            # Check if host test (obj_instance) is derived from mbed_host_tests.BaseHostTest()
+            if not isinstance(obj_instance, BaseHostTest):
+                # In theory we should always get host test objects inheriting from BaseHostTest()
+                # because loader will only load those.
+                self.logger.prn_err("host test must inherit from mbed_host_tests.BaseHostTest() class")
```

Comment:
Like I said in comment above this print:
```
# In theory we should always get host test objects inheriting from BaseHostTest()
# because loader will only load those.
```
so this print would never appear in normal life. Maybe if someone is hacking our app this will appear.